sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -268,6 +268,14 @@ class DataFrameReader private[sql](sqlContext: SQLContext) {
   }
 
   /**
+   * Loads an ORC file and returns the result as a [[DataFrame]].
+   *
```

Comment:
The one in `DataFrameWriter` isn't an example, it says that method is equivalent to `format("orc").load(path)`.